src/dialogs/quick-open/ha-quick-open-dialog.ts

```diff
@@ -0,0 +1,328 @@
+      entityId: (ev.target as any).text,
+    });
+    this.closeDialog();
+  }
+
+  protected render() {
+    if (!this._opened) {
+      return html``;
+    }
+    return html`
+      <ha-dialog open @closed=${this.closeDialog} hideActions>
+        <table class="entities">
+          <tr>
+            <th>
+              <paper-input
+                dialogInitialFocus
+                no-label-float
+                @value-changed=${this._entityFilterChanged}
+                label="${this.hass.localize(
+                  "ui.dialogs.quick_open.filter_placeholder"
+                )}"
+                type="search"
+                value=${this._commandMode
+                  ? `>${this._entityFilter}`
+                  : this._entityFilter}
+              ></paper-input>
+            </th>
+          </tr>
+          ${Object.keys(this._entities).map((text) => {
+            const item = this._entities[text];
+            const iconPath = domainIcon(item.domain);
+            return html`
+              <tr>
+                <td>
+                  <ha-icon icon=${iconPath}></ha-icon>
+                  <a
+                    href="#"
+                    @click=${item.onClick}
+                    .text=${item.text}
+                    .domain=${item.domain}
+                    alt=${this.hass.localize(
+                      "ui.panel.developer-tools.tabs.states.more_info"
+                    )}
+                    title=${this.hass.localize(
+                      "ui.panel.developer-tools.tabs.states.more_info"
+                    )}
+                    >${item.text}</a
+                  >
+                </td>
+              </tr>
+            `;
+          })}
+        </table>
+      </ha-dialog>
+    `;
+  }
+
+  static get styles() {
+    return [
+      haStyleDialog,
+      css`
+        .entities th {
+          text-align: left;
+        }
+
+        :host([rtl]) .entities th {
+          text-align: right;
+        }
+
+        .entities tr {
+          vertical-align: top;
+          direction: ltr;
+        }
+
+        .entities tr:nth-child(odd) {
+          background-color: var(--table-row-background-color, #fff);
+        }
+
+        .entities tr:nth-child(even) {
+          background-color: var(--table-row-alternative-background-color, #eee);
+        }
+        .entities td {
+          padding: 4px;
+          min-width: 200px;
+          word-break: break-word;
+        }
+        .entities ha-svg-icon {
+          --mdc-icon-size: 20px;
+          padding: 4px;
+          cursor: pointer;
+        }
+        .entities td:nth-child(3) {
+          white-space: pre-wrap;
+          word-break: break-word;
+        }
+
+        .entities a {
+          color: var(--primary-color);
+        }
+
+        .entities {
+          width: 100%;
+        }
+
+        ha-dialog {
+          --dialog-surface-position: static;
+          --dialog-content-position: static;
+          --dialog-content-padding: 0;
```

Comment:
Why did you remove the padding?

Reply:
Trying to emulate a slim quick bar look and feel

Reply:
we should stick to the material guidelines, so we should keep the normal padding

Reply:
Yeah. Also, the slim thing made more sense when I  was *just* opening a textfield without pre-populating. It looked bulky for a text field designed to be a lightweight, out-of-the-way thing. 

Now that it's pre-populating the list it's very clear you're right about needing consistent padding.

Reply:
Done. Looks like this now:
![image](https://user-images.githubusercontent.com/1480827/95121207-94f03a80-0703-11eb-8d40-f3659a9c1a05.png)

Reply:
> @donkawechico We fix the edit card dialog to the top of the page. You can check that code out

I ended up modeling it after [dialog-media-player-browse](https://github.com/home-assistant/frontend/blob/dev/src/components/media-player/dialog-media-player-browse.ts#L100)

Reply:
Use `mwc-list` for the items, and remove the odd background.

Reply:
Yep. Up and done.

Reply:
I've also updated the gif in the PR description